app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @post = Post.find(params[:id])
+    @comment = @post.comments.build(comment_params)
+    @comment.author = current_user
+
+    redirect_to user_post_path(current_user, @post) if @comment.save
+    render plain: 'Error occured while saving data'
```

Comment:
[ ] - Nice working on your Comment controller, I get the error when trying to create a comment. You need to remove the `render plain: 'Error occurred while saving data.'`